cli/README.md

```diff
@@ -65,16 +64,27 @@ The following table includes short descriptions and the general syntax for all o
 
 | Command  | Description  |
 |:---:|---|
-| `add-resource`  | Adds a resource to a service within your project in the specified stage |
-| `auth`  | Authenticate the Keptn CLI against a Keptn installation  |
-| `create`  | Create currently allows to create a project |
+| `abort`  | Aborts the execution of a sequence |
+| `add-resource`  | Adds a local resource to a service within your project in the specified stage |
+| `auth`  | Authenticates the Keptn CLI against a Keptn installation  |
+| `completion`  | Generate completion script  |
+| `configure`  | Configures one of the specified parts of Keptn  |
+| `create`  | Creates a new project, service or secret |
+| `delete`  | Deletes a project |
+| `generate`  | Generates the markdown CLI documentation or a support archive |
+| `get`  | Displays an event or Keptn entities such as project, stage, or service |
 | `help`  | Help about any command |
-| `install`  | Install Keptn on your Kubernetes cluster |
-| `onboard`  | Onboard allows to onboard a new service |
-| `send`  | Send a Keptn event in combination with the subcommand *event* |
+| `install`  | Installs Keptn on a Kubernetes cluster |
+| `pause`  | Pauses the execution of a sequence |
+| `resume`  | Resumes the execution of a sequence |
+| `send`  | Sends an event to Keptn |
+| `set`  | Sets flags of the CLI configuration |
 | `status`  | Checks the status of the CLI |
-| `uninstall`  | Uninstalls Keptn on your Kubernetes cluster |
-| `version`  | Prints the CLI version for the current context |
+| `trigger`  | Triggers the execution of an action in keptn |
+| `uninstall`  | Uninstalls Keptn from a Kubernetes cluster |
+| `update`  | Updates an existing Keptn project |
+| `upgrade`  | Upgrades Keptn on a Kubernetes cluster |
+| `version`  | Shows the version of Keptn and Keptn CLI |
 
 ## Examples: Common operations
 Use the following set of examples to help you familiarize yourself with running the commonly used `keptn` operations:

```

Comment:
```suggestion
Use the following set of examples to familiarize yourself with running the commonly used `keptn` operations:

```

Reply:
Thanks, I've made the adjustment here: 47bac16388b6e5aaaf846b536eb47cddc7a090f2